resources/zib/zib-AdministrationAgreement.xml

```diff
@@ -0,0 +1,274 @@
+  <abstract value="true" />
+  <type value="MedicationDispense" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/MedicationDispense" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="MedicationDispense">
+      <path value="MedicationDispense" />
+      <short value="AdministrationAgreement" />
+      <alias value="Toedieningsafspraak" />
+      <mapping>
+        <identity value="zib-administrationagreement-v1.0.3-2020EN" />
+        <map value="NL-CM:9.8.20132" />
+        <comment value="AdministrationAgreement" />
+      </mapping>
+    </element>
+    <element id="MedicationDispense.extension">
+      <path value="MedicationDispense.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationDispense.extension:additionalInformation">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="additionalInformation" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-AdministrationAgreement.AdditionalInformation" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:agreementReason">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="agreementReason" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-AdministrationAgreement.AgreementReason" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:administrationAgreementDateTime">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="administrationAgreementDateTime" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-AdministrationAgreement.AdministrationAgreementDateTime" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:renderedDosageInstruction">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="renderedDosageInstruction" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RenderedDosageInstruction" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUsePeriod">
+      <path value="MedicationDispense.extension" />
+      <sliceName value="periodOfUsePeriod" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval-Period" />
+      </type>
+    </element>
+    <element id="MedicationDispense.extension:periodOfUsePeriod.value[x]">
+      <path value="MedicationDispense.extension.value[x]" />
+      <short value="PeriodOfUse" />

```

Comment:
If you put a short of "PeriodOfUse / Duration" on periodOfUseDuration, shouldn't this be "PeriodOfUse / Period"?

Reply:
Jub! Thnx